bar-trender-front-end/src/components/List.js

```diff
@@ -9,21 +9,24 @@ import {
 
 const List = (props) => {
   const { establishments } = props;
-  if (!establishments.establishments || establishments.establishments.length == undefined) return <p>No repos, sorry</p>;
+  
+  // console.log(establishments.establishments,"list");
+
+  if (!establishments || establishments.length == undefined) return <p>No establishments, sorry</p>;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7167715)
```suggestion
  if (!establishments || establishments.length == undefined) {return <p>No establishments, sorry</p>;}
```